src/apis/modTools.es6.js

```diff
@@ -16,4 +16,27 @@ const approve = (apiOptions, fullname) => {
     return apiRequest(apiOptions, 'POST', 'api/approve', { body, type: 'form' });
 }
 
-export default { remove, approve }
\ No newline at end of file
+/**
+ * Set or unset a stickied post (AKA an "Annoucement").
+ * See also: https://www.reddit.com/dev/api#POST_api_set_subreddit_sticky
+ * @function
+ * @param {Object} apiOptions
+ * @param {string} fullname The fullname of the target post
+ * @param {boolean} isStickied Whether to sticky or unsticky the post
+ * @param {number?} [stickyNum] Allows for specifying the "slot" to sticky the post
```

Comment:
oooh good catch, i think the question mark is supposed to be prefixed, not postfixed 😱